src/content/post/2023/hva-er-infrastruktur-som-kode/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Hva er infrastruktur som kode (IaC)?
+# headline:
+description:
+  Ansible, Terraform, Pulumi, CloudFormation, CDK, Serverless, Helm. Hva er
+  egentlig alt det der? Finn det ut her.
+date: 2023-11-03T15:00:00
+image: ./simen-host-2023.jpg
+keywords:
+  devops, sikkerhet, menneskelig feil, deployment, "devops-tjenester",
+  distribusjon
+author:
+  - simenandre
+---
+
+- Hva? Hvorfor? Hvordan? Hvor?
+
+Det korte svaret er at _Infrastructure as Code_ eller IAC er en måte å beskrive,
+styre og provisjonere infrastruktur på. Det vil si, alt fra servere, nettverk,
+databaser og andre ting, og i dag er det kanskje mer vanlig at man provisjonerer
+disse tingene i en av de store skyleverandørene som Google Cloud, Azure og
+Amazon Web Services (AWS).
+
+Infrastruktur som kode har endret stort hvordan IT-ressurser håndteres og
```

Comment:
```suggestion
Infrastruktur som kode har endret hvordan IT-ressurser håndteres og
```

I feel like this sentence needs to be reworded if you want to add a strengthening adverb